packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -54,7 +59,8 @@ export default class ImageSelection implements EditorPlugin {
                     if (
                         mouseSelection &&
                         mouseSelection.type === SelectionRangeTypes.ImageSelection &&
-                        mouseSelection.image !== mouseTarget
+                        (!safeInstanceOf(mouseTarget, 'HTMLElement') ||
+                            !mouseTarget.contains(mouseSelection.image))
```

Comment:
What happens if the mouseTarget has more content than just image the image?

Reply:
reverted